scheduler/src/cook/kubernetes/compute_cluster.clj

```diff
@@ -293,9 +293,8 @@
         ^V1Pod pod (api/task-metadata->pod pod-namespace compute-cluster task-metadata)
         new-cook-expected-state-dict {:cook-expected-state :cook-expected-state/starting
                                       :launch-pod {:pod pod}}]
-    ; If a pod is not synthetic, cache a mapping of its instance-uuid to job-uuid in order to give all state machine passport events access to job-uuid
-    (when-not (api/synthetic-pod? pod-name)
-      (cache/put-cache! caches/instance-uuid->job-uuid identity pod-name (str (get-in task-metadata [:task-request :job :job/uuid]))))
+    ; Cache a mapping of pod-name to job-uuid in order to give all state machine passport events access to job attributes
+      (cache/put-cache! caches/pod-name->job-ent identity pod-name (get-in task-metadata [:task-request :job]))
```

Comment:
Why did we remove the `(when-not (api/synthetic-pod? pod-name)` check? The `pod-name` for synthetic pods is not an instance UUID.